docker/quickstart/docker-compose.yml

```diff
@@ -235,7 +237,8 @@ services:
       - elasticsearch
       - neo4j
     command: "sh -c 'while ping -c1 kafka-setup &>/dev/null; do echo waiting for kafka-setup... && sleep 1; done; \
-              echo kafka-setup done! && ./mae-consumer-job/bin/mae-consumer-job'"
+              echo kafka-setup done! && \
+              dockerize -wait http://neo4j:7474 -timeout 240s && java -jar mae-consumer-job.jar'"
```

Comment:
`dockerize` needs to be installed first in the Dockerfile of datahub-mae-consumer.